### PR TITLE
Update puppyinstaller

### PR DESCRIPTION
**Pull request by Kopernicusss**
add "BDRV" to install, fixes apt breakage after hard drive install.

### CODE REVIEW

File: woof-code/rootfs-skeleton/usr/sbin/puppyinstaller [hunk 1]
```diff
@@ -864,7 +865,7 @@ case $PMEDIA in usb*)
 		yaf-splash -bg orange -text "`eval_gettext \"Please wait, copying files to \\\$DESTPART ...\"`" &
 		XPID=$!
 		mkdir -p /mnt/data
-		for i in vmlinuz initrd.gz ${DISTRO_PUPPYSFS} ${DISTRO_ZDRVSFS} ${DISTRO_FDRVSFS} ${DISTRO_ADRVSFS} ${DISTRO_YDRVSFS}; do
+		for i in vmlinuz initrd.gz ${DISTRO_PUPPYSFS} ${DISTRO_ZDRVSFS} ${DISTRO_FDRVSFS} ${DISTRO_ADRVSFS} ${DISTRO_YDRVSFS} ${DISTRO_BDRVSFS}; do
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/sbin/puppyinstaller:868]: Why not align this list with https://github.com/puppylinux-woof-CE/woof-CE/blob/5ff59258c353b2f8739af7099b9b4ebd7f34b6f7/woof-code/rootfs-skeleton/usr/sbin/bootflash#L873?

Comment (peabee) [reply woof-code/rootfs-skeleton/usr/sbin/puppyinstaller:868]: Full installs are deprecated and we prefer to promote frugal installs - however this addition does seem sensible.

There is no need to update the historic Copyright comment as Github records all "blame".

Comment (Kopernicusss) [reply woof-code/rootfs-skeleton/usr/sbin/puppyinstaller:868]: patch updated to reverse copyright change.

Comment (dimkr) [reply woof-code/rootfs-skeleton/usr/sbin/puppyinstaller:868]: But you haven't added kbuild and other SFSs 🤷🏿

**Pull request merged**